example/reduce/src/reduce-main.cpp

```diff
@@ -75,7 +75,7 @@ int main()
     // REDUCE CALL:
     // Takes the arguments: accelerator device, host device, accelerator queue, size of data, pointer-like to memory,
     // reduce lambda.
-    Idx reduceResult
+    TRed reduceResult
```

Comment:
Please undo the rename changes in this file. I already totally overworked the example in PR #59. Therefore, it would only causes merge conflicts.